typings/balena-sdk.d.ts

```diff
@@ -317,6 +317,7 @@ declare namespace BalenaSdk {
 		application_environment_variable: ReverseNavigationResource<
 			ApplicationVariable
 		>;
+		build_environment_variable: ReverseNavigationResource<ApplicationVariable>;
```

Comment:
Even thought those two are the same, let's create a separate `BuildVariable` type.

Reply:
No probs